src/include/pbs_share.h

```diff
@@ -106,6 +106,11 @@
 
 #define MAX_INT_LEN 10
 
+/* Values of sched attribute 'job_run_wait' */
+#define RW_NONE "none"
```

Comment:
Just a suggestion: we should rename this to REPLY_WAIT_*  or maybe just WAIT_* and it took some time to get that RW means reply wait (generally RW stand for read-write) 😄